src/client/method-help.c

```diff
@@ -49,6 +49,10 @@ void usage_bluechi() {
         printf("    usage: daemon-reload nodename\n");
         printf("  - set-loglevel: change the log level of the bluechi-controller or a connected node\n");
         printf("    usage: set-loglevel [nodename] [loglevel]\n");
+        printf("  - get-default: get a default target of connected node\n");
+        printf("    usage: get-default [nodename]\n");
+        printf("  - set-default: set a default target of connected node\n");
+        printf("    usage: set-default [nodename] [target] [force]\n");
```

Comment:
As systemctl doesn't mention force parameter neither in manpage nor help, I'd remove the parameter from the help in bluechictl

Reply:
Changed